website/sidebars.json

```diff
@@ -115,7 +115,11 @@
         "label": "Android",
         "collapsible": false,
         "collapsed": false,
-        "items": ["headless-js-android", "signed-apk-android"]
+        "items": [
+          "headless-js-android",
+          "signed-apk-android",
+          "build-speed-android"
```

Comment:
suggest moving to "Environment Setup" and naming "Build Speed" to be more generic since if it works for both platforms when using ccache then it no longer fits for just ios or just android.

Or, perhaps split between "use a single ABI" doc just for android, and a separate generic ccache document? :thinking: 

Or, perhaps make a section for All Platforms and file it in there? Information architecture is not my strong suit. My strong suit is more git deeplinks and code snippets :sweat_smile:

Reply:
We might move this under the "Performance" section